server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/IndicateHandCount.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Free & Fair Colorado RLA System
+ * 
+ * @title ColoradoRLA
+ * @created Aug 9, 2017
+ * @copyright 2017 Free & Fair
+ * @license GNU General Public License 3.0
+ * @author Daniel M. Zimmerman <dmz@galois.com>
+ * @description A system to assist in conducting statewide risk-limiting audits.
+ */
+
+package us.freeandfair.corla.endpoint;
+
+import javax.persistence.PersistenceException;
+
+import com.google.gson.JsonSyntaxException;
+
+import spark.Request;
+import spark.Response;
+
+import us.freeandfair.corla.Main;
+import us.freeandfair.corla.model.ContestToAudit;
+import us.freeandfair.corla.model.DepartmentOfStateDashboard;
+import us.freeandfair.corla.persistence.Persistence;
+import us.freeandfair.corla.query.DepartmentOfStateDashboardQueries;
+
+/**
+ * The endpoint for selecting the contests to audit.
+ * 
+ * @author Daniel M Zimmerman
+ * @version 0.0.1
+ */
+@SuppressWarnings("PMD.AtLeastOneConstructor")
+public class IndicateHandCount extends AbstractDoSDashboardEndpoint {
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public EndpointType endpointType() {
+    return EndpointType.POST;
+  }
+  
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public String endpointName() {
+    return "/hand-count";
+  }
+
+  /**
+   * @return STATE authorization is necessary for this endpoint.
+   */
+  public AuthorizationType requiredAuthorization() {
+    return AuthorizationType.STATE;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  protected ASMEvent endpointEvent() {
+    return INDICATE_FULL_HAND_COUNT_CONTEST_EVENT;
+  }
+  
+  /**
+   * Attempts to select contests for audit. 
+   * 
+   * Session query parameters: <tt>random-seed</tt>
+   * 
+   * @param the_request The request.
+   * @param the_response The response.
+   */
+  @Override
+  public synchronized String endpoint(final Request the_request, 
+                                      final Response the_response) {
+    ok(the_response, "Contests selected");
+    // try {
```

Comment:
Yes, it was copied from another endpoint and does not actually live here, but the code we have to write will be modeled on it somewhat.